src/ages/ages.ts

```diff
@@ -114,6 +115,52 @@ export const ages: AgeConfig[] = [
       },
     ],
   },
+  {
+    ageName: "age3",
+    startTimestamp: BigNumber.from(new Date("2022-12-29T15:00:00.000Z").getTime() / 1000),
+    endTimestamp: BigNumber.from(new Date("2023-04-07T15:00:00.000Z").getTime() / 1000),
+    distribution: ageThreeDistribution,
+    subgraphUrl: "https://api.thegraph.com/subgraphs/name/morpho-labs/morpho-rewards",
+    epochs: [
+      {
+        id: "age3-epoch1",
+        number: 7,
+        epochName: "epoch1",
+        snapshotProposal: "0xf402161143c42ac5edf6589b7833eb760cbe556e18451a232e6be3d34d06f287",
+        snapshotBlock: undefined,
```

Comment:
why is it undefined?

Reply:
I think that it is because we can't know the block number yet (in the future)

Reply:
Yes but i though that e 3 epoch 1 already started, forget about it